app/views/gobierto_budgets/indicators/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<script type="text/javascript">
+  window.populateDataYear = {
```

Comment:
I think this is the problem of the JS error. You already have a year here: https://github.com/PopulateTools/gobierto/blob/master/app/views/layouts/_head.html.erb#L49, so you don't need to define this.

Reply:
@ferblape, The first thing I did as you rightly say was use window.populateData.year, but If I use the variable window.populateData.year it does not filter me, it is fixed, it is always 2015, and I need it to take the different years that I change with the year selector.

Reply:
Ok, could check the JS error in the budgets home page?

Reply:
I've fixed it. Other javascript errors appear, which are also in production.